apps/condo/domains/meter/components/TabContent/MeterReading.tsx

```diff
@@ -108,10 +107,17 @@ const MetersTableContent: React.FC<MetersTableContentProps> = ({
         fetchPolicy: 'network-only',
     })
 
-    const { breakpoints } = useLayoutContext()
     const [search, handleSearchChange, handleSearchReset] = useSearch()
     const { UpdateMeterModal, setSelectedMeter } = useUpdateMeterModal(refetch)
-    const { MultipleFiltersModal, setIsMultipleFiltersModalVisible, ResetFiltersModalButton } = useMultipleFiltersModal(filtersMeta, MeterReadingFilterTemplate, handleSearchReset)
+    const { MultipleFiltersModal, ResetFiltersModalButton, OpenFiltersButton } = useMultipleFiltersModal(
+        filtersMeta,
+        MeterReadingFilterTemplate,
+        handleSearchReset,
+        null,
+        null,
+        [],
+        { tab }
```

Comment:
I think we need to update `useMultipleFiltersModal` args to object instead many params, out of this task

Reply:
It is used only in 3 places, I will fix it in this task